client/src/features/dashboard/components/DatasetsDashboard.tsx

```diff
@@ -65,10 +65,8 @@ function OtherDatasetsButton({ totalDatasets }: OtherDatasetsButtonProps) {
         data-cy="explore-other-datasets-btn"
         className="btn btn-outline-rk-pink"
```

Comment:
For centered alignment:
```suggestion
        className={cx("btn", "btn-outline-rk-pink", "d-flex", "align-items-center")}
```

![image](https://github.com/SwissDataScienceCenter/renku-ui/assets/951086/7aec88bd-9440-490b-b1ea-1350f041424a)
![image](https://github.com/SwissDataScienceCenter/renku-ui/assets/951086/29b2db2f-fee1-4dd2-b6ca-4470a51990ca)